src/Evolution/Executables/Cce/CharacteristicExtractBase.hpp

```diff
@@ -120,4 +120,8 @@ struct CharacteristicExtractDefaults {
       StepChoosers::ErrorControl<StepChooserUse::LtsStep,
                                  evolved_coordinates_variables_tag,
                                  coord_vars_selector>>;
+
+  using ccm_psi0 = tmpl::list<
```

Comment:
Why does CCE need CCM? This seems like a poor design decision. CCE shouldn't need to know about CCM

Reply:
My current plan is to make `CharacteristicExtractDefaults` in `src/Evolution/Executables/Cce/CharacteristicExtractBase.hpp` a base metavariable class for both CCM and CCE, templated on a `bool` parameter `UsesPartiallyFlatCartesianCoordinates`  or `EvolveCCM` (or whatever we want to call it). The normal CCE's metavariable inherits `CharacteristicExtractDefaults<false>`, which is actually what we are doing in `src/Evolution/Executables/Cce/CharacteristicExtract.hpp`. For my current CCM's metavariable, it inherits `CharacteristicExtractDefaults<true>`. This won't break CCE because here CCM simply needs a few more tags to mutate.

For `ccm_psi0`, the easiest way I could think of is to make it empty when not using CCM. Let me know if you don't like it or have a better idea.